qa/suites/rados/monthrash/ceph.yaml

```diff
@@ -8,8 +8,12 @@ overrides:
         mon osdmap full prune min: 15
         mon osdmap full prune interval: 2
         mon osdmap full prune txsize: 2
+        mon scrub inject crc mismatch: 0.01
+        mon scrub inject missing keys: 0.05
 # thrashing monitors may make mgr have trouble w/ its keepalive
-    log-whitelist:
+    log-ignorelist:
```

Comment:
nautilus doesn't have ignore-list

Reply:
Damn, reversed the conflict. Thanks @jdurgin